backend/chromeApiServer/src/main/java/com/canvas/service/helperServices/CanvasClientService.java

```diff
@@ -43,14 +42,14 @@ public class CanvasClientService {
     private static final Logger logger = LoggerFactory.getLogger(CanvasClientService.class);
 
 
-    /**
-     * Constructor with not arguments
-     */
-    public CanvasClientService() {
+    @Autowired
+    protected CanvasClientService(Environment env) {
+        this.canvasUrl = env.getProperty("canvas.url");
         this.okHttpClient = new OkHttpClient();
     }
 
-    protected CanvasClientService(OkHttpClient okHttpClient) {
+    protected CanvasClientService(String canvasUrl, OkHttpClient okHttpClient) {
```

Comment:
Still missing `@Autowired` before the method declaration

Reply:
Since there can only be one @autowired constructor this cannot be @autowired too.